src/AspNet.Security.OAuth.Weibo/WeiboAppBuilderExtensions.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using Microsoft.Extensions.Options;
+using AspNet.Security.OAuth.Weibo;
+using JetBrains.Annotations;
+
+namespace Microsoft.AspNetCore.Builder
+{
+    public static class WeiboAppBuilderExtensions
```

Comment:
This doesn't match the convention we use for the other provider. E.g https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/blob/dev/src/AspNet.Security.OAuth.GitHub/GitHubAuthenticationExtensions.cs#L17